feat(#2092): document recipient resolving multiple recipients (#2093)
medic/cht-docs: content/en/building/reference/app-settings/_index.md

| *valid phone number* | requested phone number |

> [!NOTE]

> > - when recipient is an array of multiple recipients, it will try to resolve first existing recipient from an array.

Reviewer: ```suggestion
> - when recipient is an array, the first resolved recipient from the array will be selected. 
```
